app/routes/upload.step3.tsx

```diff
@@ -1,31 +1,43 @@
 import { LoaderFunctionArgs } from '@remix-run/node'
 import { useLoaderData } from '@remix-run/react'
-import { formCookie } from '~/lib/cookies'
+import CVDisplay from '~/components/CVDisplay'
+import { cvCookie, enhancedCvCookie, jobDetailsCookie } from '~/lib/cookies'
 
 export const loader = async ({ request }: LoaderFunctionArgs) => {
   const cookieHeader = request.headers.get('Cookie')
-  const cookieData = await formCookie.parse(cookieHeader)
 
-  return { cookieData }
+  const extractedCV = await cvCookie.parse(cookieHeader)
+  const jobData = await jobDetailsCookie.parse(cookieHeader)
+  const enhancedCv = await enhancedCvCookie.parse(cookieHeader)
+
+  return { extractedCV, jobData, enhancedCv }
 }
 
 const step3 = () => {
   // @ts-ignore
-  const { cookieData } = useLoaderData()
+  const { extractedCV, jobData, enhancedCv } = useLoaderData()
```

Comment:
**nitpick**: Remove console.log statements to clean up the code before production.